src/components/Section/Identification/Identification.jsx

```diff
@@ -1,349 +1,90 @@
+ */
 
-          <hr className="section-divider" />
-          <ApplicantBirthDate
-            name="birthdate"
-            {...this.props.ApplicantBirthDate}
-            section="identification"
-            subsection="birthdate"
-            dispatch={this.props.dispatch}
-            onUpdate={this.handleUpdate.bind(this, 'ApplicantBirthDate')}
-            onError={this.handleError}
-            required={true}
-            scrollIntoView={false}
-          />
+class Identification extends React.Component {
+  render() {
+    const subsection = this.props.subsection || 'intro'
 
-          <hr className="section-divider" />
-          <ApplicantBirthPlace
-            name="birthplace"
-            {...this.props.ApplicantBirthPlace}
-            section="identification"
-            subsection="birthplace"
-            dispatch={this.props.dispatch}
-            onUpdate={this.handleUpdate.bind(this, 'ApplicantBirthPlace')}
-            onError={this.handleError}
-            required={true}
-            scrollIntoView={false}
-          />
+    const subsectionClasses = `view view-${subsection || 'unknown'}`
 
-          <hr className="section-divider" />
-          <ApplicantSSN
-            name="ssn"
-            {...this.props.ApplicantSSN}
-            section="identification"
-            subsection="ssn"
-            dispatch={this.props.dispatch}
-            onUpdate={this.handleUpdate.bind(this, 'ApplicantSSN')}
-            onError={this.handleError}
-            required={true}
-            scrollIntoView={false}
-          />
+    const title = subsection === 'review' && i18n.t('review.title')
+    const para = subsection === 'review' && i18n.m('review.para')
+    const showTop = subsection === 'review'
 
-          <hr className="section-divider" />
-          <OtherNames
-            name="othernames"
-            {...this.props.OtherNames}
-            section="identification"
-            subsection="othernames"
-            defaultState={false}
-            dispatch={this.props.dispatch}
-            onUpdate={this.handleUpdate.bind(this, 'OtherNames')}
-            onError={this.handleError}
-            required={true}
-            scrollIntoView={false}
-          />
+    /** TODO - this should come from Redux store */
+    const formType = 'SF86'
 
-          <hr className="section-divider" />
-          <ContactInformation
-            name="contacts"
-            {...this.props.Contacts}
-            section="identification"
-            subsection="contacts"
-            minimumPhoneNumbers={1}
-            minimumEmails={1}
-            shouldFilterEmptyItems={true}
-            defaultState={false}
-            dispatch={this.props.dispatch}
-            onUpdate={this.handleUpdate.bind(this, 'Contacts')}
-            onError={this.handleError}
-            required={true}
-            scrollIntoView={false}
-          />
+    const navButtons = (
```

Comment:
Is there a reason why we need to abstract this component into its own variable?

Reply:
The only reason I did this was because there's a check for whether these buttons exist when rendering the `top-btns` div (see https://github.com/18F/e-QIP-prototype/pull/1431/files/128717208eec1861d840c27c4340026bbd6db792#diff-e0df189d16e0e8e1b2360e055bb7bd57R56). I'm not sure why this check exists because I don't see how the two are related, and it also seems that `navButtons` will *always* exist, but I left it in there for parity 🤷‍♀️ happy to remove it if we determine it's unnecessary.

Reply:
Yeah, it looks like `navButtons` will _always_ evaluate as truthy because it's a React component. Removing `navButtons` from the conditional is probably safe.
```
 {showTop && (
   <div className="top-btns"><ErrorList /></div>
  )}
```

Then there's no need for the `navButtons` variable.

Reply:
well if both `back` and `next` are false-y then the component does return `null`, but I don't see why that should make any difference to the `top-btns` div. I'll remove it.

Reply:
If I remember correctly, even if the component returns `null`, `navButtons` is still a React component.  The `render` shouldn't affect whether it's truthy or falsey.